### ISSUE TITLE
Flaw in app

### ISSUE THREAD

**Issue by eldhosejohn** [wt65074/mouxebackend#2]
There is a flaw in the app, can u get back to me at my twitter id - eldhojohn

**Comment by wt65074** [comment]
Yeah. I know it doesn't work anymore and I have been meaning to remove it from github.

**Comment by eldhosejohn** [comment]
not that, there are some keys which are compromised. Please remove them

https://github.com/wt65074/mouxebackend/compare/ba3ff5f5a3...8c3f08712a#diff-9ff7a9b76e17b84fe29b1c4e77a76672L7
